### DISCUSSION TITLE
Demo of integration with Wails

### DISCUSSION THREAD

**Discussion by panda-madness** [php/frankenphp#593] [Ideas]
[Wails](https://wails.io) is a Go based tool to build desktop applications using native platform WebView components. For curiosity's sake I've tried to compile a basic Wails app with FrankkenPHP as the request handler, and I've been able to get a basic Laravel welcome page to display successfully. [Here's the demo repository](https://github.com/panda-madness/wails-frankenphp).

I have minimal Go knowledge, so I can't really explore this idea much further, maybe someone more experienced can pick this up and figure out if this integration is worth working on.

On the surface this seems interesting as a potential alternative way to build desktop apps,  kind of like what NativePHP is trying to do.

**Comment by withinboredom** [comment]
I... am speechless. In a ~~terrified~~ good way.

**Comment by crazywhalecc** [comment]
FrankenPHP is a web server, and I think integrating php-embed and wails is a better way.

**Reply by dunglas** [reply]
FrankenPHP - the Go library - can be used without needing a web server. I think that it's what we're talking about in this context.
